storage/innobase/xtrabackup/doc/source/version-check.rst

```diff
@@ -0,0 +1,72 @@
+.. _version-check:
+
+================================================================================
+Version Checking
+================================================================================
+
+Complex software projects (such as MySQL flavors) always have versions with
+minor or major compatibility problems, regressions and bugs, etc. It is too
+difficult to take care about all these compatibility nuances for a human, and
+this is the reason for a |version-check| feature to exist.
+
+The feature allows compatible tools to query the Version Check database and
+print advice on any potential risks associated with any of the relevant
+software versions.
+
+Usage
+-----
+
+*Version Check* was implemented in |pt| 2.1.4, and was enabled by default in
```

Comment:
Speaking of xtrabackup, version check has been added in 2.1.5 and was always enabled by default.